package/piksi_system_daemon/src/main.c

```diff
@@ -191,6 +211,37 @@ int main(void)
   SETTING_NOTIFY("ethernet", "netmask", eth_netmask, TYPE_STRING, eth_ip_config_notify);
   SETTING_NOTIFY("ethernet", "gateway", eth_gateway, TYPE_STRING, eth_ip_config_notify);
 
+  char name_string[64];
+  if (file_read_string("/img_tbl/boot/name",
+                       name_string, sizeof(name_string)) == 0) {
+    /* sw_build_id contains the full image name */
+    static char sw_build_id[sizeof(name_string)];
+    strncpy(sw_build_id, name_string, sizeof(sw_build_id));
+    READ_ONLY_PARAMETER("system_info", "sw_build_id", sw_build_id, TYPE_STRING);
+
+    /* sw_version contains everything before the git hash */
+    static char sw_version[sizeof(name_string)];
+    strncpy(sw_version, name_string, sizeof(sw_version));
+    char *sep = strstr(sw_version, "-g");
+    if (sep != NULL) {
+      *sep = 0;
+    }
+    READ_ONLY_PARAMETER("system_info", "sw_version", sw_version, TYPE_STRING);
```

Comment:
can we actually rename this to "firmware_version." to avoid confusion?

Then, in piksi_firmware_private we will rename the version and date settings to:
- gnss version
- gnss date